src/routers/StackRouter.js

```diff
@@ -110,15 +110,25 @@ export default (
           action.type === NavigationActions.NAVIGATE &&
           childRouters[action.routeName] !== undefined
         ) {
+          const childRouter = childRouters[action.routeName];
+          let childState;
+          if (childRouter !== null) {
+            const childAction =
+              action.action ||
+              NavigationActions.init({ params: action.params });
```

Comment:
I dont think we should pass params through here. The params were specified on this top-level action. If you want to specify params on the sub-action, you can always manually specify the `action.action`